File: easy/tickets/tickets.kt
fun main(args: Array<String>){

	println("Number of points: ")
    var points = "500"!!.toInt()
	println("cost of the squirt gun: ")
    var cost = "40"!!.toInt()

    if(points/12 >= cost){
    	println("Buy it!")
    }
    else{
    	println("Try again")
    }


}

File: easy/tickets/tickets.rb
"""
@author	André Peralta
@version 20/05/2021
"""

def buy n_tickets, price

	n_points = n_tickets/12

	if n_points >= price
		return "Buy it!"
	else
		return "Try again"
	end

end

puts "Number of tickets: "
n_tickets = gets.to_i
puts "Cost of the squirt gun: "
price = gets.to_i

puts buy(n_tickets,price)

